lib/routes.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:nativemoon/screens/Home/index.dart';
+import 'package:nativemoon/screens/Login/index.dart';
+
+class Routes {
+
+  var routes = <String, WidgetBuilder>{
+    "/Home": (BuildContext context) => new HomePage()
+  };
+
+  Routes() {
+    runApp(new MaterialApp(
+      title: "Flutter Flat App",
```

Comment:
Not sure if this title is relevant at all, but maybe something more akin to nativemoon? Once again, not sure if relevant.

Reply:
It's not relevant for the build of the app but makes sense to change it to nativemoon.